input/collector_platform.go

```diff
@@ -0,0 +1,36 @@
+package input
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/pganalyze/collector/state"
+	"github.com/shirou/gopsutil/host"
+)
+
+// TODO: is there a better place to initialize this?
+var collectorStartTime = time.Now()
+
+func getCollectorPlatform() state.CollectorPlatform {
+	hostInfo, err := host.Info()
+	if err != nil {
+		// TODO: log this? return error?
```

Comment:
What should we do with errors here?

Reply:
Maybe `logger.PrintVerbose` (and possibly only with test runs?), so we could debug this if we're surprised by this not working in some case.